utils/strings.go

```diff
@@ -60,17 +58,3 @@ func TrimRight(s string, cutset byte) string {
 	}
 	return s[:lenStr]
 }
-
-// EqualFold the equivalent of strings.EqualFold
```

Comment:
Please keep the function, it might be used by some users, because we export these

Reply:
kept it and fixed a bug 👍🏼